netbench-driver/src/alloc.rs

```diff
@@ -15,6 +15,12 @@ impl Allocator {
     }
 }
 
+impl Default for Allocator {
```

Comment:
Perhaps a tiny bit outside of the scope of this PR, but it would be great to have a comment describing why we are using a different allocator.

Reply:
Yea I am not sure, this was simply covering some clippy warnings.